app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,91 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductBadRequestException;
+import com.codesoom.assignment.UserBadRequestException;
+import com.codesoom.assignment.UserNotFoundException;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserData;
+import com.github.dozermapper.core.DozerBeanMapperBuilder;
+import com.github.dozermapper.core.Mapper;
+import org.springframework.stereotype.Service;
+
+/**
+ * 사용자에 대한 요청을 수행한다.
+ */
+@Service
+public class UserService {
+    private final UserRepository userRepository;
+    private final Mapper mapper = DozerBeanMapperBuilder.buildDefault();
+
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    /**
+     * 주어진 식별자 해당하는 사용자를 리턴한다.
```

Comment:
조사가 빠졌습니다. JavaDoc의 summary description은 API 문서 웹 페이지로 빌드될 것을 생각하고 가능한 한 잘 작성해야 합니다.
```suggestion
     * 주어진 식별자에 해당하는 사용자를 리턴한다.
```

Reply:
네 명심하겠습니다
https://github.com/CodeSoom/spring-week5-assignment-1/pull/10#discussion_r584255954